src/sidebar/services/test/groups-test.js

```diff
@@ -114,6 +114,15 @@ describe('groups', function() {
       });
     });
 
+    it('always sends the `expand` parameter', function () {
+      const svc = service();
+      return svc.load().then(() => {
+        const call = fakeApi.groups.list.getCall(0);
+        assert.isObject(call.args[0]);
+        assert.equal(call.args[0].expand, 'organization');
```

Comment:
This is fine. We sometimes use `assert.match(someObject, sinon.match({ /* partial object */}))` to verify that a subset of an object's properties match expected values.